blueprint-no-op/build.gradle.kts

```diff
@@ -30,51 +91,16 @@ android {
         targetCompatibility = JavaVersion.VERSION_17
     }
     kotlin {
-        jvmToolchain(17)
         explicitApi()
     }
-    kotlinOptions {
-        jvmTarget = "17"
-        freeCompilerArgs = listOf("-opt-in=com.popovanton0.blueprint.ExperimentalBlueprintApi")
-    }
-
-    buildFeatures.compose = true
-    composeOptions.kotlinCompilerExtensionVersion = libs.versions.composeCompiler.get()
 }
 
 tasks.named("assemble") {
     dependsOn("check")
 }
 
-dependencies {
-    implementation(libs.androidx.compose.ui.util)
-    implementation(libs.androidx.compose.ui)
-    implementation(libs.androidx.compose.foundation)
-    implementation(libs.androidx.annotation)
-
-    debugImplementation(libs.androidx.compose.ui.tooling)
-    debugImplementation(libs.androidx.compose.ui.testManifest)
-
-    testImplementation(libs.junit)
-    implementation(libs.androidx.compose.ui.tooling)
-    testImplementation(libs.androidx.compose.material)
-    testImplementation(libs.testParameterInjector)
-    testImplementation(libs.equalsverifier)
-
-    androidTestImplementation(libs.androidx.testJunit)
-    androidTestImplementation(libs.androidx.compose.ui.testJunit)
-}
+group = "com.popovanton0.blueprint"
+version = libs.versions.blueprint.get()
 
 publishing {
-    publications {
-        register<MavenPublication>("release") {
-            groupId = "com.popovanton0.blueprint"
-            artifactId = "blueprint-no-op"
-            version = libs.versions.blueprint.get()
-
-            afterEvaluate {
-                from(components["release"])
-            }
-        }
-    }
-}
```

Comment:
Do you think it will be possible to publish Blueprint KMP on jitpack? Is there a way to test this before merging this PR?

Reply:
I believe the best way would be for you to pull to a separate branch and try to publish from there. [jitpack docs](https://docs.jitpack.io/faq/) do mention that you can build from a branch by requesting a version in the form of `<branchName>-SNAPSHOT`.

Do note that I've tried to do that with my fork and got nothing (UPD: okay, that's not entirely true, there's a log with build failure due to JS support: https://jitpack.io/#sfalexrog/blueprint)

Truth be told, I'd be very surprised if jitpack actually produced usable KMP artifacts - the publication seems to be a multi-artifact one, somewhat similar to how Gradle plugins are published (an empty artifact with a .pom/.module and actual, platform-specific artifacts)

Reply:
Update: I've disabled JS and the jitpack build eventually got through: https://jitpack.io/com/github/sfalexrog/blueprint/kmp-2d615f34e7-1/build.log

I then tried to use the jitpacked library (artifact `com.github.sfalexrog.blueprint:blueprint:kmp-SNAPSHOT`) instead of the project and it... just worked? I'm extremely impressed!

So, chances are, the library is going to just work in KMP projects, and we don't need to add anything else to the build